app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -109,7 +109,10 @@ def fetch_jwt_claims_to_validate
       def initialize_validate_restrictions
         @restriction_validator = Authentication::AuthnJwt::ValidateRestrictionsOneToOne
         @restrictions_from_annotations_class = Authentication::ResourceRestrictions::GetServiceSpecificRestrictionsFromAnnotation
```

Comment:
this can be done in a separate effort but param names should match their class. And also command classes should be with `.new` in the dependencies